.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,34 @@
+## Description
+<!--
```

Comment:
Should there be a section which is for linking issues which the PR closes? And describes how to do it so GH actually links it.

```md
## Closes the following issues:
<!--
Closes #XXXXX, 
Fixes #XXXXX

For items in other repositories, for example in Godot-Proposals:
Closes godot/godot-proposals/#XXXXXX
-->
```